__tests__/app.test.js

```diff
@@ -82,6 +82,67 @@ describe("/api/articles/:article_id", () => {
         expect(message).toBe("Bad request");
       });
   });
+  test("PATCH:200 will update the votes of an article using an inc_votes object, increasing the votes by the value of the inc_votes key", () => {
+    const newVotes = { inc_votes: 5 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(newVotes)
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article.votes).toBe(105);
+      });
+  });
+  test("PATCH:200 will update the votes of an article if the increse in votes is negative, representing a decrease in votes)", () => {
+    const newVotes = { inc_votes: -5 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(newVotes)
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article.votes).toBe(95);
+      });
+  });
+  test("PATCH:404 sends an appropriate status and error message when given a valid but non-existent article_id", () => {
+    const newVotes = { inc_votes: -5 };
+    return request(app)
+      .patch("/api/articles/999")
+      .send(newVotes)
+      .expect(404)
+      .then(({ body: { message } }) => {
+        expect(message).toBe("article_id not found");
+      });
+  });
+  test("PATCH:400 sends an appropriate status and error message when given an invalid article_id", () => {
+    const newVotes = { inc_votes: -5 };
+    return request(app)
+      .patch("/api/articles/destroyeverything")
+      .send(newVotes)
+      .expect(400)
+      .then(({ body: { message } }) => {
+        expect(message).toBe("Bad request");
+      });
+  });
+  test("PATCH:200 if the article does not have a votes key, the article will gain one with a value equal to the new votes", () => {
+    const newVotes = { inc_votes: 10 };
+    return request(app)
+      .patch("/api/articles/2")
+      .send(newVotes)
+      .expect(200)
+      .then(({ body: { article } }) => {
+        console.log(article);
+        expect(article.votes).toBe(10);
+      });
+  });
```

Comment:
The articles *should* all have a votes key, so this test shouldn't be necessary 🤔